### ISSUE TITLE
Cannot do `enableUTF8' in Term::ReadLine::Gnu

### ISSUE THREAD

**Issue by eserte** [dod38fr/config-model#11]
Some of my smoker systems report test failures:
```
Cannot do `enableUTF8' in Term::ReadLine::Gnu at /home/cpansand/.cpan/build/2016110818/Config-Model-2.093-N_eWnQ/blib/lib/Config/Model/TermUI.pm line 167.
# Looks like your test exited with 255 just after 3.
t/term_ui.t ....................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 10/13 subtests 
```
Probably this happens if Term::ReadLine::Gnu is installed, but too old.

**Comment by dod38fr** [comment]
You're probably right. `enableUTF8` was added to `Term::ReadLine::Gnu` v1.32 released on 2016-06-06. 

I'll add the dependency version.

Thanks for the report.

**Comment by dod38fr** [comment]
Actually, I'll test whether `enableUTF8` is available (with `$t->can('enableUTF8`)`). This will avoid headaches when `Term::ReadLine::Perl` is loaded by `Term::ReadLine`.

**Comment by dod38fr** [comment]
Fixed in v2.094.

Thanks
